cmd/image.go

```diff
@@ -79,7 +83,7 @@ func auditImages(image string, items Items) (results []Result) {
 		containers, result := containerIter(item)
 		for _, container := range containers {
 			checkImage(container, image, result)
-			if result != nil && result.err > 0 {
+			if result != nil && result.err >= 0 {
```

Comment:
In order to get the `INFO` messages we need to look for `err` **`>=`** `0`.

Reply:
I am not sure about this since in the past we have only printed error messages i.e. faulty configurations. But I agree it is debatable. @klautcomputing

Reply:
Well this kind of doesn't make sense... `result.err` will always be `>= 0`(well should be, even though  in the code it's an `int` we should treat it as a `unit` and maybe change the code) so the check is redundant. Which means it was there for a reason because we only want to log errors.

But maybe we should either:
- have 2 different `log` methods? or
- have a `result.info` because this is not an error case.
if we want to allow non errors to be printed as well.

This question here is a bit hypothetical because no one but you uses the `image` programmatically but still I think we should come up with a way of handling more than the error case inside kubeaudit and find a real solution for this.

Reply:
I have a way out for this one. Following more Linux style error handling like a big enum of errors that we support and every function should use from it.  Lots of wins for us by doing that:

- Centralized error tracking
- Adding more semantics to error rather than some bunch of number

Reply:
Regarding that INFO that we should have a flag and then we will print INFO in all the cases. So that we have a unified interface everywhere